core/lib/refinery/core/engine.rb

```diff
@@ -63,7 +63,8 @@ def refinery_inclusion!
         end
       end
 
-      initializer "refinery.routes", :after => :set_routes_reloader_hook do |app|
+      # FIXME: Find why in Rails 5, if we use `:set_routes_reloader_hook` it redirect sprockets assets engines paths to 404
```

Comment:
I need your expertise to understand the changes between Rails 4 and 5. But in all cases the changes, fix our problem